util/test/demos/vk/vk_shader_editing.cpp

```diff
@@ -181,6 +241,26 @@ void main()
 
       FinishUsingBackbuffer(cmd, VK_ACCESS_TRANSFER_WRITE_BIT, VK_IMAGE_LAYOUT_GENERAL);
 
+      // Fill bufin with 111
+      vkCmdFillBuffer(cmd, bufin.buffer, 0, 1024, 111);
+      // Fill bufout with 222
+      vkCmdFillBuffer(cmd, bufout.buffer, 0, 1024, 222);
+      vkh::cmdPipelineBarrier(
+          cmd, {},
+          {
+              vkh::BufferMemoryBarrier(VK_ACCESS_TRANSFER_WRITE_BIT, VK_ACCESS_SHADER_WRITE_BIT,
+                                       bufout.buffer, 0, 1024),
```

Comment:
`bufin` here should really have a barrier from `VK_ACCESS_TRANSFER_WRITE_BIT` to `VK_ACCESS_SHADER_READ_BIT`.